docs/source/CompilationFlowVerification.ipynb

```diff
@@ -143,34 +100,26 @@
    "source": [
     "Check out the [reference documentation](library/VerifyCompilation.rst#mqt.qcec.verify_compilation) for more information."
    ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": []
   }
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
+   "display_name": "Python 3",
    "language": "python",
    "name": "python3"
   },
   "language_info": {
    "codemirror_mode": {
     "name": "ipython",
-    "version": 3
+    "version": 2
    },
    "file_extension": ".py",
    "mimetype": "text/x-python",
    "name": "python",
    "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython3",
-   "version": "3.9.15"
+   "pygments_lexer": "ipython2"
```

Comment:
I don't know why, but this change looks strange to me. Can it be that some Python2 slipped in there somehow?

Reply:
Shouldn't be the case. Only let the notebook cleaner run over it

Reply:
Really don't know how the hell that happened, needed to manually change it back...